lib/modifiers.js

```diff
@@ -45,7 +45,19 @@ function nestedRestore (arr) {
   const length = arr.length
   for (var i = 0; i < length; i++) {
     const { key, target, value } = arr[i]
-    target[key] = value
+    if (has(target, key)) {
+      target[key] = value
+    }
+    /* istanbul ignore else */
+    if (typeof target === 'object') {
+      const targetKeys = Object.keys(target)
+      targetKeys.forEach((tKey) => {
```

Comment:
use a for(;;) loop instead, it's way faster.

Reply:
OK :)

Reply:
done